include/jemalloc/internal/tsd_internals.h

```diff
@@ -63,7 +63,9 @@ typedef ql_elm(tsd_t) tsd_link_t;
 #define TSD_DATA_SLOW							\
     O(tcache_enabled,		bool,			bool)		\
     O(reentrancy_level,		int8_t,			int8_t)		\
+    O(thread_tcache_max,	size_t,			size_t)		\
```

Comment:
The order of this matters. We should put the `size_t` to after the thread_nhbins (which can be `uint16_t` instead?).

Reply:
Switching the order can make the size down to 2656B. Under the new layout, changing `thread_nhbins` to `uint16_t` or not does not change the size. FYI, `nhbins` was `unsigned`, yet `uint16_t` should be more than enough. I think I will go ahead to make it `uint16_t` unless there is some hidden history behind this.